src/main/java/com/google/devtools/build/lib/rules/repository/BindRule.java

```diff
@@ -61,6 +61,10 @@ public Metadata getMetadata() {
 }
 /*<!-- #BLAZE_RULE (NAME = bind, TYPE = OTHER, FAMILY = Workspace)[GENERIC_RULE] -->
 
+<em><p>Warning: use of <code>bind()</code> is not recommended. See "<a
+href="https://github.com/bazelbuild/bazel/issues/1952">Consider removing bind</a>" for a long
+discussion of its issues and alternatives.</p></em>
```

Comment:
Thanks for the comment. I modelled this on the [best practices](https://docs.bazel.build/versions/master/best-practices.html#repository-rules). Maybe we could add the following text to both:

>Use repository names (eg `@com_google_protobuf//:protobuf`) or, if necessary, [alias()](https://docs.bazel.build/versions/master/be/general.html#alias) instead.

Is that suitable? It would be good to better explain when `alias()` is recommended/necessary, but I don't understand the use cases, and #1952 doesn't discuss it much. When is it useful?